src/downshift.js

```diff
@@ -266,13 +266,17 @@ class Downshift extends Component {
         isOpen: false,
       },
       () => {
-        const inputNode = this._rootNode.querySelector(`#${this.inputId}`)
-        inputNode && inputNode.focus && inputNode.focus()
+        this.focusInput()
         cbToCb(cb)()
       },
     )
   }
 
+  focusInput = () => {
```

Comment:
This can actually be a regular method because we don't pass it as a callback to anything: `focusInput() {`